### PR TITLE
Update README for new APIs

### PR DESCRIPTION
**Pull request by camphillips22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,65 +16,91 @@ SQL recipes for Data Export analysis are in the [Data Export Cookbook](https://g
...
+specified duration.
+The default is 1 hour, but if a different file size is desirable, this can be modified to meet your specific needs. The
+max duration is 24 hours.
 
-`hauser` can safely be stopped and restarted. For Redshift and BigQuery, it uses the `SyncTable` to keep track of what export files have been processed, and will restart from the last known sync point.
+#### `ExportDelay`
+Determines how long to wait before creating an export.
+This delay is necessary because there is some latency between when an event is recorded and when it is available and complete.
+24 hours is the default, but is also fairly conservative. In many cases this can be reduced safely to 3 hours, but note that
+events from "[swan songs]" may not be available.
```
Comment (claygoddard) [inline README.md:40]: `swan songs` can it link to the reference below?

File: README.md [hunk 2]
```diff
@@ -13,65 +16,91 @@ SQL recipes for Data Export analysis are in the [Data Export Cookbook](https://g
...
 
-### Redshift Notes
-To use the Redshift warehouse, set the `Warehouse` config option to `redshift`.
+#### `StartTime`
+Determines the date-time that should be used a starting point for creating the exports.
```
Comment (claygoddard) [inline README.md:43]: Does this have a hyphen normally?
```suggestion
Determines the datetime that should be used a starting point for creating the exports.
```

**Review by claygoddard — APPROVED** [review]
LGTM! (just one nit)

File: README.md (hunk 1 above)
Comment (camphillips22) [reply README.md:40]: Yes. You could check by viewing the rich diff, but it's also part of the spec and is called a [shortcut reference link](https://github.github.com/gfm/#shortcut-reference-link)

File: README.md (hunk 2 above)
Comment (camphillips22) [reply README.md:43]: Not sure why I wrote it this way 😄

**Pull request merged**